src/main/java/mil/dds/anet/database/PositionDao.java

```diff
@@ -225,10 +230,11 @@ public int setPersonInPosition(String personUuid, String positionUuid) {
           .execute();
 
       getDbHandle()
-          .createUpdate(
-              "/* positionSetPerson.remove1 */ UPDATE positions set \"currentPersonUuid\" = NULL "
-                  + "WHERE \"currentPersonUuid\" = :personUuid")
-          .bind("personUuid", personUuid).execute();
+          .createUpdate("/* positionSetPerson.remove1 */ UPDATE positions "
+              + "SET \"currentPersonUuid\" = NULL, type = :type, \"updatedAt\" = :updatedAt "
+              + "WHERE \"currentPersonUuid\" = :personUuid")
+          .bind("personUuid", personUuid).bind("type", DaoUtils.getEnumId(revokePrivilege(currPos)))
+          .bind("updatedAt", DaoUtils.asLocalDateTime(now)).execute();
```

Comment:
I think it would be better to write in order; 
1. type = :type
2. updatedAt = :updatedAt 
3. currentPersonUuid = :personUuid
So;
bind("type", DaoUtils.getEnumId(revokePrivilege(currPos)))
.bind("updatedAt", DaoUtils.asLocalDateTime(now))
.bind("personUuid", personUuid).execute();